### ISSUE TITLE
A link has no accessible name, innerText or title

### ISSUE THREAD

**Issue by ollie-iterators** [CityOfBoston/Iterators#203]
Screengrab/videos (insert file names here)
2020 11 10, Link has no accessible name, innerText or title.png
![2020 11 10, Link has no accessible name, innerText or title](https://user-images.githubusercontent.com/46904826/98725786-97c5f700-2363-11eb-9b30-75f4c849f4a8.png)

Link
https://d8-stg.boston.gov/news/boston-expands-bike-share-east-boston

Node # (if known)
1741

Type of User:  Anonymous

Environment
(Device > OS > Browser)
Computer > MacOS 10.15.7 > Chrome 86

Trusted Tester Test
Links

Brief Description
A link has no accessible name, innerText or title

Steps to reproduce
1)	Load the URL
2)	Go down to the link in the “About Hubsway” section that is named “Boston Bikes website” and check out the link directly to the right of that link

Desired Result
•	All links have an accessible name, innerText, or a title

Actual Result
•	There is a link that has no accessible name, innerText or title on the page.

**Comment by subaha-cob** [comment]
Took out an empty link going to https://www.bostonbikes.org/. 

Updated on boston.gov

**Comment by ollie-iterators** [comment]
This issue appears to be fixed
